package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "0.1.0",
+  "version": "1.1.0",
```

Comment:
In the [Semver syntax](https://semver.org/) this should change to `1.0.0`. However, I spoke with @dr3 IRL and we've agreed that this should have already been `1.0.0` so it makes sense for this PR to bump to `1.1.0`. 

Adding this for future reference of npm versions.